### PR TITLE
Now only removing accept header from PUT requests as per spec

### PR DESCRIPTION
**Pull request by msk-**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mojaloop/central-services-shared",
-  "version": "7.4.8",
+  "version": "7.4.9",
```
Comment (elnyry) [inline package.json]: dont want to block this now, but this should be 7.5.0 now

Comment (msk-) [reply package.json]: Not a big deal. I'll update it.

Comment (elnyry) [reply package.json]: Thanks

**Comment by msk-** [comment]
Closed due to prejudice against this PR from CircleCI. Duplicate opened #91.

**Pull request closed without merging**